crml/transaction-payment/src/constants.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018-2020 Parity Technologies (UK) Ltd. and Centrality Investments Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! # Transaction Payment Module
+//!
+//! Transaction Payment Customized Error Code Constants
+
+pub mod error_code {
+	// Core to asset exchange errors
+	pub const CORE_TO_ASSET_PRICE_NOT_ABOVE_ZERO: u8 = 190;
```

Comment:
I know it's approved and merged. However, I believe it's good to still share our suggestions. Here, I think enum is better suited for encompassing all the possible error cases.

Reply:
Yes I agree with you on this. But the reason why `u8` was used was because we have to propagate `InvalidTransaction::Custom(u8)` error type